CHANGELOG.md

```diff
@@ -1,3 +1,15 @@
+# dbt_zendesk v0.14.0
+
+## Bug Fixes
+- The `int_zendesk__schedule_spine` model was updated to properly account for schedules that recognized daylight savings time (DST) at one point in time, and then stopped recognizing it at a later date. ([PR #128](https://github.com/fivetran/dbt_zendesk/pull/128))
+  - For example, the Hong Kong timezone originally recognized DST, but them stopped in 1979. The previous versions of this package only recorded the schedule business hours until 1979. This update addresses this bug.
+
+## Under the Hood
+- Included auto-releaser GitHub Actions workflow to automate future releases. ([PR #128](https://github.com/fivetran/dbt_zendesk/pull/128))
```

Comment:
should we include the updated maintainer PR template? or is it too under the hood...

Reply:
I think that is a good think to include. Will update and thanks for calling out.

Reply:
Updated.